### DISCUSSION TITLE
Flipper Zero

### DISCUSSION THREAD

**Discussion by cybermbl** [community/community#119824] [Programming Help]
### Select Topic Area

Question

### Body

I just purchased a Flipper Zero. I uploaded the latest firmware from the qflipper website. Downloaded app on my iPhone and was able to pair via Bluetooth. But now I get a preload fail error for every firmware app. Anyone have any suggestions? 
![image](https://github.com/community/community/assets/167231524/5db259aa-8079-4010-b020-76652bdb121f)

**Comment by ebndev** [comment]
Thanks for posting in the GitHub Community, @cybermbl! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Best of luck!
